### PR TITLE
Remove leftover cruft from about page b25d8aa9d6bf52e8e7c2f5ac8484dd7…

### PR DESCRIPTION
**Pull request by mejackreed**
…4197c83b9 that was never taken out when move to about page gem was added af9dc24ba7cb8c721906946bcb69621f2e9cd025

### CODE REVIEW

**Comment by jmartin-sul** [comment]
rebase and force push and i'll merge?

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -8,6 +8,6 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += ['about.css.sass', 'registration.css', 'report.css', '*.js', '*.coffee']
+Rails.application.config.assets.precompile += ['registration.css', 'report.css', '*.js', '*.coffee']
```
Comment (jmartin-sul) [inline config/initializers/assets.rb]: don't care.  up @mejackreed whether to change this.

Comment (atz) [inline config/initializers/assets.rb]: Hound is annoying on this.  I guess it doesn't have a knowledge of rubocop defaults, forcing us to state them explicitly?

Comment (mejackreed) [inline config/initializers/assets.rb]: Yep, I've done this in another project before. Thoughtbot provides their own default rubocop.yml https://github.com/thoughtbot/hound/blob/master/config/style_guides/ruby.yml

Comment (mejackreed) [inline config/initializers/assets.rb]: Override the default setting provided by Hound: 
https://github.com/sul-dlss/sul-embed/blob/master/.rubocop.yml#L11-L13

Comment (jmartin-sul) [inline config/initializers/assets.rb]: :+1:

**Pull request merged**